releases/release-5.0.2.md

```diff
@@ -0,0 +1,58 @@
+---
+title: TiDB 5.0.2 Release Notes
+---
+
+# TiDB 5.0.2 Release Notes
+
+发版日期：2021 年 6 月 9 日
+
+TiDB 版本：5.0.2
+
+## 兼容性更改
+
+## 新功能
+
+## 提升改进
+
++ TiDB
+
+    - 优化了统计信息读取过程。如果缓存的统计数据是最新的，则跳过读取 mysql.stats_histograms 表，来避免频繁的数据读取 [#24317](https://github.com/pingcap/tidb/pull/24317)
+
++ TiFlash
+
+    - 优化 apply region snapshots 操作的内存占用
+    - 优化锁操作以避免 DDL 和读数据相互阻塞
+    - 支持 integer 和 real 类型转化为 real 类型
+
+## Bug 修复
+
++ TiDB
+
+    - 修复了在某些情况下，使用前缀索引和 Index Join 导致的 Panic 的问题 [#24824](https://github.com/pingcap/tidb/pull/24824)
+    - 修复了 point get 的 prepare plan cache 被事务中的 point get 语句不正确使用的问题 [#24765](https://github.com/pingcap/tidb/pull/24765)
+    - 修复了当 collation 为 ascii_bin/latin1_bin 时，写入错误前缀索引的值的问题 [#24680](https://github.com/pingcap/tidb/pull/24680)
+    - 修复了正在执行的事务被 gc worker 中断的问题 [#24652](https://github.com/pingcap/tidb/pull/24652)
+    - 修复了当 `new-collation` 开启且 `new-row-format` 关闭的情况下，点查在聚簇索引下可能出错的问题 [#24611](https://github.com/pingcap/tidb/pull/24611)
+    - Refactor converting partition keys for shuffle hash join [#24490](https://github.com/pingcap/tidb/pull/24490)
```

Comment:
@wshwsh12 This item is missing the translation. Please help translate this one. Thanks!

Reply:
```suggestion
    - 为 Shuffle Hash Join 重构分区键的转换功能 [#24490](https://github.com/pingcap/tidb/pull/24490)
```